lib/intercom/errors.rb

```diff
@@ -81,6 +81,8 @@ class Intercom::AttributeNotSetError < IntercomError; end
   # Raised when unexpected nil returned from server
   class Intercom::HttpError < IntercomError; end
 
+  class ApiVersionInvalid < IntercomError; end
```

Comment:
I know this is self-explanatory, but I would add a line above like in all other exceptions declared in this file to keep it consistent

Reply:
Added in db3a03a